### DISCUSSION TITLE
Using MDsveX (👍18)

### DISCUSSION THREAD

**Discussion by o-t-w** [11ty/eleventy#3989] [Enhancement Queue]
I would like to be able to include Svelte components within my Markdown files using [MDsveX](https://github.com/pngwn/MDsveX). Is this possible with 11ty?

**Comment by jevets** [comment]
@o-t-w 

You could try adding plugins to `markdown-it` or setting your own markdown library instance
https://www.11ty.io/docs/languages/markdown/

Aside from that, I don't think you'll be able to work directly with `.svelte` files in Eleventy quite yet (as far as Eleventy compiling them), at least not until custom template file extensions and handlers/engines are available.

**Comment by zachleat** [comment]
Yeah that’s at #117. Currently the top issue in the enhancement queue. Please upvote it!

**Comment by zachleat** [comment]
This repository is now using [lodash style issue management](https://twitter.com/samselikoff/status/991395669016436736) for enhancements. This means enhancement issues will now be closed instead of leaving them open.

View [the enhancement backlog here](https://github.com/11ty/eleventy/issues?utf8=%E2%9C%93&q=label%3Aneeds-votes+sort%3Areactions-%2B1-desc+). Don’t forget to upvote the top comment with 👍!

**Comment by zachleat** [comment]
(Also putting this issue in the Enhancement queue, thank you!)

**Comment by o-t-w** [comment]
Putting this here as reference of another popular static site generators approach https://github.com/gohugoio/hugo/issues/5894
